pkg/bugzilla/bugzilla.go

```diff
@@ -0,0 +1,170 @@
+package bugzilla
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/golang/glog"
+	"k8s.io/test-infra/prow/bugzilla"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+type BugzillaVerifier struct {
+	bzClient     bugzilla.Client
+	ghClient     github.Client
+	pluginConfig *plugins.Configuration
+}
+
+func NewBugzillaVerifier(bzClient bugzilla.Client, ghClient github.Client, pluginConfig *plugins.Configuration) *BugzillaVerifier {
```

Comment:
Godoc here and on the struct describing what the behavior of the verifier is (in rough high level).